examples/website/i3s/components/tool-bar.js

```diff
@@ -29,21 +28,22 @@ const Container = styled.div`
 const ToolButton = styled.button`
   background: ${props => props.active ? '#4F52CC' : 'transparent'};
   color: ${props => props.active ? 'white' : 'rgba(255, 255 , 255, .8)'};
-  border-radius: 4px;
-  height: 80%;
-  margin-left: 2px;
   cursor: pointer;
   border: none;
   font-size: 22px;
-  min-width: 40px;
-  flex: 1 0 1px;
-  padding: 0;
+  flex: 1 1 0px;
+  margin: 2px 0 2px 1px;
+  height: 56px;
+  width: 16.5vw;
   &:hover {
     transition: all 1s;
     color: white;
```

Comment:
To avoid hover effect on mobile while click on button.
Also animation is removed.
```suggestion
  @media (hover: hover) and (pointer: fine) {
     &:hover { color: white; }
   }
```